src/main/java/constants/UrlConstants.java

```diff
@@ -0,0 +1,6 @@
+package constants;
+
+public class UrlConstants {
+    public static final String URI = "/oauth/token";
+    public static final String URL = "http://localhost:49000/users";
```

Comment:
move to properties, and please introduce some profiling logic so that I can choose which profile to choose: local or remote, like we did in web training